pkgs/applications/emulators/craftos-pc/default.nix

```
@@ -14,29 +14,30 @@
 }:
 
 let
+  version = "2.7.3";
   craftos2-lua = fetchFromGitHub {
     owner = "MCJack123";
     repo = "craftos2-lua";
-    rev = "v2.6.6";
-    sha256 = "cCXH1GTRqJQ57/6sWIxik366YBx/ii3nzQwx4YpEh1w=";
+    rev = "v${version}";
+    sha256 = "sha256-g0cYbAKAK8SEdjTID7FFus9rncfSPYEUfuQaatu9F0s=";
   };
   craftos2-rom = fetchFromGitHub {
     owner = "McJack123";
     repo = "craftos2-rom";
-    rev = "v2.6.6";
-    sha256 = "VzIqvf83k121DxuH5zgZfFS9smipDonyqqhVgj2kgYw=";
+    rev = "v${version}";
+    sha256 = "sha256-UaachE3lonHrNaOPrM3oAIQ8Kq0NuryytS6/sypF6fw=";
```

```suggestion
    sha256 = "sha256-t76Yltx7vHNoAAFvNpYLKuwFja4On6M20upmG6w3C1M=";
```

You didn't generate the hashes correct. Did you maybe use some prefetch tool with the wrong arguments.

---

Yep, fixed all the SHAs in the latest push.